setup.py

```diff
@@ -49,17 +49,16 @@ def read(path: str) -> str:
     package_data={"crate.operator": ["data/*"]},
     setup_requires=["setuptools>=58", "setuptools_scm>=6.2"],
     install_requires=[
-        "aiopg==1.4.0",
-        "bitmath==1.3.3.1",
-        "kopf==1.35.6",
+        "aiopg<1.5",
```

Comment:
Due to there not being a lockfile for setuptools, I am strongly against pinning version ranges - this eventually leads to some library releasing a patch version that breaks backwards compatibility accidentally and you no longer have predictable builds. Let's leave it like it was please.

Reply:
I've removed the pinning of version ranges again.